frontend/package.json

```diff
@@ -1,13 +1,16 @@
 {
   "name": "frontend",
   "version": "0.1.0",
+  "homepage": "/class-notes",
   "private": true,
   "dependencies": {
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "antd": "^5.0.5",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.4.5",
     "react-scripts": "5.0.1",
```

Comment:
put ^ before version for backward compatibility

Reply:
Done